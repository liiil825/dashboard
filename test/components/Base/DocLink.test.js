import React from 'react';
import { render } from 'enzyme';
import toJson from 'enzyme-to-json';

import DocLink from 'components/Base/DocLink';

describe('Base/DocLink', () => {
  it('basic render', () => {
    const wrapper = render(
      <DocLink name="STEPPER_FOOTER_CREATE_APP_2">
        《Helm 规范及应用开发》
      </DocLink>
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
