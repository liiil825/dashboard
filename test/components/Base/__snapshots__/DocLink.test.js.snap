// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base/DocLink basic render 1`] = `
<a
  class=""
  href="https://docs.openpitrix.io/v0.3/zh-CN/developer-guide/helm-specification/"
  rel="noopener noreferrer"
  target="_blank"
>
  《Helm 规范及应用开发》
</a>
`;
